### DISCUSSION TITLE
Access PGPT on network

### DISCUSSION THREAD

**Discussion by MLEdelen1** [imartinez/privateGPT#1780] [General]
How would I access PrivateGPT on all computers within my network? Like on another computer use the internal IP of 192.168.1.???:8001 and have access?

**Comment by Bobota2006** [comment]
you can do that by running privategpt normally with whatever config you want and then go to the browser and search for the ip address of the computer that running it followed by the port for example:192.168.1.1:8001. This works on any browser on any platform (I think)

**Comment by atakan-tr** [comment]
This method didn't work for me. Does anyone have a different idea?

**Comment by williamthe5thc** [comment]
This also did not work for me, I have added the allow_origins to match the IP of the host computer, but can't connect to the computer except via local host....
